src/battle/battle.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+/**
+ * @brief A function that takes you to battle
```

Comment:
Please be a bit more descriptive. With these comments, one still needs to go to the implementation of the function to see what it does. A simple "it starts a static battle between a main character with default stats and a hand-written goblin creature, and compares their HP to decide the winner" should be enough for this iteration.

Reply:
done